Lib/importlib/_bootstrap_external.py

```diff
@@ -259,6 +258,7 @@ def cache_from_source(path, debug_override=None, *, optimization=None):
 
     """
     if debug_override is not None:
+        import _warnings
```

Comment:
`_warnings` is a built-in module, so this does not make a performance difference. As well, this module is very unique in needing to control what modules it imports as it can **only** import built-in modules, so having them together for visibility/documentation is important. As such, please undo the changes to this file.